### PR TITLE
Bob debugging doc

### PR DESCRIPTION
**Pull request by otsakir**
How to debug Bob using IntelliJ - doc

## PR checklist

* [ ] Code
	* [ ] Add engine and/or editor unit tests.
	* [ ] New and changed code follows the overall code style of existing code
	* [ ] Add comments where needed
* [ ] Documentation
	* [ ] Make sure that API documentation is updated in code comments
	* [x] Make sure that manuals are updated (in github.com/defold/doc)
* [ ] Prepare pull request and affected issue for automatic release notes generator
	* [ ] Pull request - Write a message that explains what this pull request does. What was the problem? How was it solved? What are the changes to APIs or the new APIs introduced? This message will be used in the generated release notes. Make sure it is well written and understandable for a user of Defold.
	* [ ] Pull request - Write a pull request title that in a sentence summarises what the pull request does. Do not include "Issue-1234 ..." in the title. This text will be used in the generated release notes.
	* [ ] Pull request - Link the pull request to the issue(s) it is closing. Use on of the [approved closing keywords](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue).
	* [ ] Affected issue - Assign the issue to a project. Do not assign the pull request to a project if there is an issue which the pull request closes.
	* [ ] Affected issue - Assign the "breaking change" label to the issue if introducing a breaking change.
	* [ ] Affected issue - Assign the "skip release notes" is the issue should not be included in the generated release notes.

### CODE REVIEW

File: com.dynamo.cr/com.dynamo.cr.bob/README.md [hunk 1]
```diff
@@ -18,3 +18,39 @@ Caching of libraries is based on the Git SHA1, to the actual commit
...
+A _Run configuration_ has to be specified that will execute bob.jar. Set it up as if executing it on the command line.
+
+1. From the _Configurations_ dropdown on the top bar (next to the hammer) click _Edit configuration_ to open the _Run/Debug Configurations_ dialog.
+
+2. Click _Add new/JAR Application_ and set a proper name for it.
+
+3. In the _Configuration_ tab use the following settings:
+
+	* `Path to JAR` - Absolute path to bob.jar. For example, /home/.../defold/com.dynamo.cr/com.dynamo.cr.bob/dist/bob.jar
+	* `Program arguments` - Command line arguments given when executing bob. For example, `--platform x86_64-linux distclean build --archive bundle --variant debug`. It really depends on what parts of Bob's functionallity you want to debug.
```
Comment (britzl) [inline com.dynamo.cr/com.dynamo.cr.bob/README.md:44]: functionallity -> functionality

**Review by britzl — CHANGES_REQUESTED** [review]
Looks good! Very useful. I found a typo and a format issue.

File: com.dynamo.cr/com.dynamo.cr.bob/README.md (hunk 1 above)
Comment (otsakir) [reply com.dynamo.cr/com.dynamo.cr.bob/README.md]: ok

File: com.dynamo.cr/com.dynamo.cr.bob/README.md [hunk 2]
```diff
@@ -18,3 +18,39 @@ Caching of libraries is based on the Git SHA1, to the actual commit
 * Note that the SHA1, in general, is not identical to the requested version as the requested version can be
   symbolic, e..g HEAD, 1.0, etc. Moreover, for tags, the underlying SHA1 to the actual commit in question is used
   and not the SHA1 for the tag-object.
+
+Debugging
+---------
+
+Once bob.jar has been built, it can be debugged with IntelliJ IDE (Community IntelliJ IDEA 2022.3.1 was used in this guide).
+
+1. Start the IDE and on the initial welcome screen choose _Open_' project. Navigate to Bob's source directory `com.dynamo.cr/com.dynamo.cr.bob` inside Defold's source tree.
```
[the remark this answers is not in the document]
Comment (otsakir) [reply com.dynamo.cr/com.dynamo.cr.bob/README.md]: ok

**Review by britzl — APPROVED** [review]

**Pull request merged**